.github/ISSUE_TEMPLATE/site_update.md

```diff
@@ -0,0 +1,21 @@
+---
+name: 🖥 Site Update
+about: Suggest an update to the Service Mesh Patterns website
+title: '[Site]'
+labels: 'area/website'
```

Comment:
```suggestion
labels: 'area/website', 'framework/reactjs'
```

Reply:
Opting for Gatsby. Seems better for a multi-page static site

Reply:
Oh, ok, @Nikhil-Ladha. 👍